src/addScore.js

```diff
@@ -1,37 +1,41 @@
-import {
-  form, user, score, apiURL,
-} from './variable.js';
+import { form, apiURL } from "./variable.js";
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your readme summary more detailed to make your readme more descriptive and professional 👍